CHANGELOG

```diff
@@ -1,3 +1,7 @@
+2015-09-23: v0.3.0
+  Support Mesos 0.25 NetworkInfo as an IPSource (#266, #267, #268)
```

Comment:
There might be other Mesos 0.25 things that we still need to do so we can't say that.